frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseRebaseBranchOntoParentAction.java

```diff
@@ -62,15 +62,14 @@ public void update(AnActionEvent anActionEvent) {
     if (state != Repository.State.NORMAL) {
 
       var stateName = Match(state).of(
-          Case($(Repository.State.GRAFTING), "during an ongoing cherry-pick"),
-          Case($(Repository.State.DETACHED), "in the detached head state"),
-          Case($(Repository.State.MERGING), "during an ongoing merge"),
-          Case($(Repository.State.REBASING), "during an ongoing rebase"),
-          Case($(Repository.State.REVERTING), "during an ongoing revert"),
+          Case($(Repository.State.GRAFTING), "ongoing cherry-pick or revert"),
+          Case($(Repository.State.DETACHED), "detached head state"),
```

Comment:
```suggestion
          Case($(Repository.State.DETACHED), "detached head"),
```
is now enough

Reply:
reverted to previous version of messages